assets/sass/main.scss

```diff
@@ -55,3 +55,6 @@
 html {
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
 }
+
+// Add your own rules in $CONVOS_HOME/assets/sass/custom.css
+@import "http://local/custom/asset/sass/custom.scss";;
```

Comment:
Could you also extend https://github.com/Nordaaker/convos/blob/master/t/web-custom-assets.t ?

Just add something like this:

```
my $custom = File::Spec->catfile($ENV{CONVOS_HOME}, qw(assets sass _custom.scss));
# open and write to $custom, and then test that the custom styles are included with something like this:
->content_like(qr{.convos-message.same-user .title});
```